cms/djangoapps/contentstore/views/component.py

```diff
@@ -41,23 +41,9 @@
 # NOTE: This list is disjoint from ADVANCED_COMPONENT_TYPES
 COMPONENT_TYPES = ['discussion', 'html', 'problem', 'video']
 
-<<<<<<< HEAD
 # Constants for determining if these components should be enabled for this course
-SPLIT_TEST_COMPONENT_TYPE = 'split_test'
-OPEN_ENDED_COMPONENT_TYPES = ["combinedopenended", "peergrading"]
-NOTE_COMPONENT_TYPES = ['notes']
-
-if settings.FEATURES.get('ALLOW_ALL_ADVANCED_COMPONENTS'):
-    ADVANCED_COMPONENT_TYPES = sorted(set(name for name, class_ in XBlock.load_classes()) - set(COMPONENT_TYPES))
-else:
-    ADVANCED_COMPONENT_TYPES = settings.ADVANCED_COMPONENT_TYPES
 XBLOCKS_ALWAYS_IN_STUDIO = getattr(settings, 'XBLOCKS_ALWAYS_IN_STUDIO', [])
-
-ADVANCED_COMPONENT_CATEGORY = 'advanced'
-ADVANCED_COMPONENT_POLICY_KEY = 'advanced_modules'
-=======
 ADVANCED_COMPONENT_TYPES = sorted(set(name for name, class_ in XBlock.load_classes()) - set(COMPONENT_TYPES))
```

Comment:
I think this means we no longer need to manually maintain the `ADVANCED_COMPONENT_TYPES` setting in config/secrets.